cdap-api/src/main/java/co/cask/cdap/api/app/AbstractApplication.java

```diff
@@ -178,9 +179,15 @@ protected void scheduleWorkflow(Schedule schedule, String workflowName, Map<Stri
    * @param scheduleName the name of the schedule
    * @param workflowName the name of the Workflow
    *
-   * @return The {@link ScheduleConfigurer} used to configure the schedule
+   * @return The {@link ScheduleBuilder} used to configure the schedule
    */
-  protected ScheduleConfigurer configureWorkflowSchedule(String scheduleName, String workflowName) {
-    return configurer.configureWorkflowSchedule(scheduleName, workflowName);
+  // TODO: javadocs
+  protected ScheduleBuilder buildSchedule(String scheduleName, String workflowName) {
+    return configurer.buildSchedule(scheduleName, workflowName);
+  }
+
+  // TODO: javadocs
+  protected void schedule(ScheduleCreationSpec scheduleCreationSpec) {
+    configurer.schedule(scheduleCreationSpec);
```

Comment:
Yea, `ScheduleCreationSpec` is the new one. It's just what we want to call the return value of the object returned from the builder.